core/capabilities/launcher.go

```
@@ -484,6 +491,12 @@ func (w *launcher) exposeCapabilities(ctx context.Context, myPeerID p2ptypes.Pee
 			return fmt.Errorf("could not unmarshal capability config for id %s", cid)
 		}
 
+		methodConfig := capabilityConfig.CapabilityMethodConfig
+		if methodConfig != nil { // v2 capability
+			w.exposeCapabilityV2(ctx, cid, methodConfig, myPeerID, don, idsToDONs)
```

you are missing the potential error when exposing the capability, if not will fail silently, right?